backend/migrations/000001_init_db.up.sql

```diff
@@ -71,6 +71,8 @@ INSERT INTO voting_strategies (key, name, description)
 VALUES ('token-weighted-default', 'Token Weighted', 'Number of tokens held directly corresponds to the weight of the vote');
 INSERT INTO voting_strategies (key, name, description)
 VALUES ('token-weighted-capped', 'Token Weighted - Capped', 'Number of tokens held directly corresponds to the weight of the vote, up to the specified cap.');
+INSERT INTO voting_strategies (key, name, description)
+VALUES ('one-address-one-vote', 'One Address One Vote', 'one address is simply only allowed one vote, assets do not come into play.');
```

Comment:
Should this be in a new migration instead of altering an existing?

Reply:
hmm I'm not sure, if I don't insert this strategy into the DB the tests wont pass.

Reply:
I could add it into a separate migration file ... not sure what our pattern should be for this.

Reply:
yeah it should be added as a separate migration, using the process outlined in the `README`

because when we `migrate up` on prod, only new migrations will be run.